### PR TITLE
Fix digits incorrect delimiter

### PR DESCRIPTION
**Pull request by alter369**

### CODE REVIEW

File: src/checks/incorrect_delimiter.rs [hunk 1]
```diff
@@ -16,7 +16,11 @@ impl Default for IncorrectDelimiterChecker {
 impl Check for IncorrectDelimiterChecker {
     fn run(&mut self, line: LineEntry) -> Option<Warning> {
         let key = line.get_key()?;
-        if key.trim().chars().any(|c| !c.is_alphabetic() && c != '_') {
+        if key
+            .trim()
+            .chars()
+            .any(|c| !c.is_alphabetic() && !c.is_numeric() && c != '_')
```
Comment (mgrachev) [inline src/checks/incorrect_delimiter.rs]: Rust has the method [is_alphanumeric](https://doc.rust-lang.org/std/primitive.char.html#method.is_alphanumeric) which uses these methods under the hood. Can we use it?

**Comment by mgrachev** [comment]
@alter369 👋Thank you for your contribution! 🚀

I've left a comment. Can you fix it, as well as add this PR to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md)?

File: src/checks/incorrect_delimiter.rs (hunk 1 above)
Comment (alter369) [reply src/checks/incorrect_delimiter.rs]: Sure 👌🏻

**Comment by mgrachev** [comment]
@alter369 Thanks a lot! 👍

**Pull request merged**